### PR TITLE
Crystal 0.35 support

### LINKED ISSUE #2: Crystal 0.35 support
It looks like it doesn't compile on Crystal 0.35. The redis shard is outdated, and some Time methods changed.

### PR DESCRIPTION
**Pull request by jwoertink**
Fixes #2 

This PR lets the shard compile on Crystal 0.35. However, there's still a few issues to sort out.

* [x] - Specs fail locally (and on the CI apparently)
* [x] - The logger needs to be refactored to support the new logger
* [ ] - ....
* [ ] - Profit!

### CODE REVIEW

**Comment by jwoertink** [comment]
@fernandes I'm not sure what specs were broken before this PR. Would you like them fixed in this PR, or a separate one? Also same with the logger? Right now it's just deprecation warnings, but should they be fixed in this, or a separate PR?

File: spec/cable/handler_spec.cr [hunk 1]
```diff
@@ -143,7 +143,7 @@ describe Cable::Handler do
       request = HTTP::Request.new("GET", "/unknown_route?test_token=1", headers)
 
       io_with_context = create_ws_request_and_return_io_and_context(handler, request)[0]
-      io_with_context.to_s.should eq("")
```
Comment (jwoertink) [inline spec/cable/handler_spec.cr:146]: Not sure if this is meant to be nothing, but when the server hits a route that doesn't exist, we get back the 404.

File: src/cable/handler.cr [hunk 2]
```diff
@@ -37,12 +37,13 @@ module Cable
...
         socket.on_close do
           connection.close
-          Cable::Logger.info "Finished \"#{path}\" [WebSocket] for #{remote_address} at #{Time.now.to_s}"
+          Cable::Logger.info "Finished \"#{path}\" [WebSocket] for #{remote_address} at #{Time.utc.to_s}"
         end
       end
 
       Cable::Logger.info "Successfully upgraded to WebSocket (REQUEST_METHOD: GET, HTTP_CONNECTION: Upgrade, HTTP_UPGRADE: websocket)"
       content = ws.call(context)
+      content.as(Proc(IO, Nil)).call(context.response.output)
```
Comment (jwoertink) [inline src/cable/handler.cr:46]: I'm not sure if this is needed, but without it, the websocket block is never called (at least on this Fiber). By adding it, the `socket.send` welcome is added to the response.

One other thing is that the type of `ws.call(context)` kept coming back as 

```
In src/cable/handler.cr:48:21

 48 | raise content.call.inspect
                    ^---
Error: undefined method 'call' for Array(Log::Entry) (compile-time type is (Array(Log::Entry) | Bool | IO | Int32 | Int64 | Proc(IO, Nil) | Nil))
```

so I had to typecast it just to get this to run.

**Comment by fernandes** [comment]
@jwoertink I added the fix from #9 and fixed CI, let's see if it's gonna be ✅ now!

**Pull request merged**